.flake8

```
@@ -60,14 +60,32 @@ max-line-length = 100
 
 # Allow certain violations in certain files:
 per-file-ignores =
-  docs/conf.py: WPS221, WPS226, WPS301, WPS305, WPS323, WPS420, WPS429, WPS433
+  # The following ignores have been researched and should be considered permanent
+  # each should be preceeded with an explanation of each of the error codes
+  # These might be commented out as they overlap with the perfile ignores
+  # once the per file ignores are removed or added within the file itself
+  # they should be able to be uncommented
+
+  # S101: Allow the use of assert within the tests directory
+  # tests/*: S101
+
+  # The following were present during the initial implemetation
+  # Please try to keep it in alphbetical order to make finding files easier for the next person
```

// I wonder if codespell can lint this
```suggestion
  # Please try to keep it in alphabetical order to make finding files easier for the next person
```